website/content/docs/nia/tasks.mdx

```diff
@@ -44,7 +44,9 @@ Below are details on the types of execution conditions that Consul-Terraform-Syn
 
 The services condition is the default condition to execute a task. Tasks with the services condition monitor and execute on either changes to a list of configured services or changes to any services that match a given regex.
 
-In order to configure a task with the services condition, the list of services to trigger the task must be configured in the task's [`services`](/docs/nia/configuration#services) or a regex must be configured in the task's `condition` block. Only one of these two options can be configured for a single task. See the [Services Condition](/docs/nia/configuration#services-condition) configuration section for more details.
+There are two ways to configure a task with a services condition. Only one of the two options below can be configured for a single task:
+1. Configure a task's [`services`](/docs/nia/configuration#services) field to specify the list of services to trigger the task
```

Comment:
Should we say that this is deprecated?

Reply:
Yes, thanks for thinking about this. I'm actually planning to open a separate PR to do one sweep to mark things as deprecated. I'm planning to save it for then.

Reply:
Sounds good!